projects/memapGui/src/main/java/fortiss/gui/listeners/action/ExitWindowListener.java

```diff
@@ -4,11 +4,13 @@
 import java.awt.event.WindowEvent;
 
 import fortiss.gui.commands.SaveAsCommand;
+import fortiss.gui.commands.SaveCommand;
 
 public class ExitWindowListener extends WindowAdapter {
 
 	@Override
 	public void windowClosing(WindowEvent e) {
-		new SaveAsCommand().execute();
+		//new SaveAsCommand().execute();
+		new SaveCommand().execute();
```

Comment:
We can keep any of the two options here. The other line can be safely removed.

Reply:
I fine with any, but it is faster with the second one (less clicking)